Xamarin.Forms.Platform.UAP/SwitchRenderer.cs

```diff
@@ -53,5 +69,54 @@ void UpdateFlowDirection()
 		{
 			Control.UpdateFlowDirection(Element);
 		}
+
+		void UpdateOnColor()
+		{
+			if (Control == null)
+				return;
+
+			var grid = Control.GetChildren<Windows.UI.Xaml.Controls.Grid>().FirstOrDefault();
+			var groups = Windows.UI.Xaml.VisualStateManager.GetVisualStateGroups(grid);
+			foreach (var group in groups)
+			{
+				if (group.Name != "CommonStates")
+					continue;
+
+				foreach (var state in group.States)
+				{
+					if (state.Name != "PointerOver")
+						continue;
+
+					foreach (var timeline in state.Storyboard.Children.OfType<ObjectAnimationUsingKeyFrames>())
+					{
+						var property = Storyboard.GetTargetProperty(timeline);
+						var target = Storyboard.GetTargetName(timeline);
+						if ((target == "SwitchKnobBounds") && (property == "Fill"))
+						{
+							var frame = timeline.KeyFrames.First();
+
+							if (_originalOnColor == null)
```

Comment:
Had to save the original value(s), because `ClearValue` would cause the control to disappear

Reply:
that's how it's done elsewhere